salt/modules/zypper.py

```diff
@@ -637,8 +636,7 @@ def install(name=None,
     # Split the targets into batches of 500 packages each, so that
     # the maximal length of the command line is not broken
     while targets:
-        cmd = cmd_install
-        cmd.extend(targets[:500])
+        cmd = cmd_install + targets[:500]
```

Comment:
I know. In fact I don't like calling `zypper` and list manipulations at all (I wrote repository management via python library (https://github.com/saltstack/salt/pull/14158) which you have replaced). So anyway it's a dirty solution IMHO.